<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cesar Cipher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Encrypted / Decrypted Message:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit_processed">
        <property name="cursor" stdset="0">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
font: 9pt &quot;Consolas&quot;;
color: rgb(0, 255, 0);</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="clear_encrypted_button">
          <property name="text">
           <string>Clear Encryption</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="select_all_encrypted_button">
          <property name="text">
           <string>Select All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="copy_encrypted_button">
          <property name="text">
           <string>Copy Encryption</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Your Message here:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit_insrted">
        <property name="cursor" stdset="0">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
font: 9pt &quot;Consolas&quot;;
color: rgb(0, 255, 0);</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="clear_inserted_button">
          <property name="text">
           <string>Clear Text</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="select_all_text_button">
          <property name="text">
           <string>Select All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="copy_text_button">
          <property name="text">
           <string>Copy Text</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="radioButton_encrypt">
          <property name="text">
           <string>Encrypt</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_decrypt">
          <property name="text">
           <string>Decrypt</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_key">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Encryption Key:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_encryption_key"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="ed_button">
        <property name="text">
         <string>Encrypt</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>plainTextEdit_insrted</tabstop>
  <tabstop>copy_encrypted_button</tabstop>
  <tabstop>clear_inserted_button</tabstop>
  <tabstop>copy_text_button</tabstop>
  <tabstop>clear_encrypted_button</tabstop>
  <tabstop>radioButton_decrypt</tabstop>
  <tabstop>lineEdit_encryption_key</tabstop>
  <tabstop>plainTextEdit_processed</tabstop>
  <tabstop>radioButton_encrypt</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>clear_inserted_button</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit_insrted</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copy_text_button</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit_insrted</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_encrypted_button</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit_processed</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copy_encrypted_button</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit_processed</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>select_all_text_button</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit_insrted</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>select_all_encrypted_button</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit_processed</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
